p4runtime_sh/shell.py

```diff
@@ -2219,8 +2219,7 @@ def _from_msg(self, msg):
             self.add(r.egress_port, r.instance)
 
     def read(self, function=None):
-        """Generate a P4Runtime Read RPC to read a single MulticastGroupEntry
-        (wildcard reads not supported).
+        """Generate a P4Runtime Read RPC to read MulticastGroupEntry.
```

Comment:
maybe use a docstring more consistent with other `read()` methods (for other entity types)

Reply:
ping